src/intl.ts

```diff
@@ -451,8 +451,8 @@ export const sectionNames = defineMessages({
     description: "page types section name",
   },
   pages: {
-    id: "H6NsC1",
-    defaultMessage: "Pages",
+    id: "0Bw3x9",
+    defaultMessage: "Content",
```

Comment:
**issue (blocking)**: when using `sectionNames.pages`, which is in top-level `intl.ts` file I expect the same string as the name suggests. Proposed change diverges from the convention.

Changing this string has also consequences for usage in whole dashboard, where this string is used, e.g. in navigation view displayed entities are now "collections, categories, content". Is this intentional?